bundles/CoreBundle/Resources/config/services.yml

```diff
@@ -192,6 +192,9 @@ services:
 
     Pimcore\Translation\Escaper\Xliff12Escaper: ~
 
+    Pimcore\DataObject\ClassesManager:
```

Comment:
This needs to be changed as well.

Reply:
ah damn it, sorry was in hurry, I removed the service declaration. There should be no need for it.

Reply:
Sure we need a service definition, but the name should be according to the fully qualified class name: `Pimcore\Model\DataObject\ClassDefinition\ClassDefinitionManager`

Reply:
Do we? The autowiring seems to work well for it.